lib/commands/deploynodeapp.js

```diff
@@ -495,14 +804,16 @@ function handleUploadResult(err, req, fileName, itemDone) {
 
 // Create a target endpoint that references the Node.js script
 function createTarget(opts, request, done) {
-  var targetDoc = mustache.render(
```

Comment:
Correct, the original implementation didn't change. This is a weird diff, see this [earlier commit](https://github.com/apigee-internal/apigeetool-node/pull/1/commits/51a08e62247ecca0a72da40b29584d2985e32dcc#diff-ffe42770ff03abad8572eb60ce558c80L696) that has the removal of a control structure in this function for template rendering.